### PR TITLE
Alright it's fixed

### PR DESCRIPTION
**Pull request by GhostEcho**
Quickly before it's deployed

### CODE REVIEW

**Comment by ChronicPwnage** [comment]
yes plz fix

**Comment by Light-Weave** [comment]
needs speedmerging before the server gets updated tbh

**Comment by improvedname** [comment]
Theres no point. Just bring this issue to the deployer rhi aand nappist only they can update the server after all

File: code/game/machinery/deployable.dm [hunk 1]
```diff
@@ -52,9 +52,9 @@
...
 			return 1
-		//var/obj/item/projectile/proj = mover
-		//if(proj.firer && Adjacent(proj.firer))
-			//return 1
+		var/obj/item/projectile/proj = mover
+		if(proj.firer && Adjacent(proj.firer))
+			return 1
 		if(prob(proj_pass_rate))
 			return 1
 		/* /obj/item/restraints/legcuffs/bola/energy/throw_impact(atom/hit_atom)
```
Comment (UndeadNappist) [inline code/game/machinery/deployable.dm]: Why is this commented out?

**Pull request merged**